app/controllers/posts_controller.rb

```diff
@@ -25,9 +25,19 @@ def edit
   # POST /posts
   def create
     @post = Post.new(post_params)
-
     if @post.save
-      redirect_to @post, notice: 'Post was successfully created.'
+      tag_params.split(",").each do |tag_new|
+        allready_exist = false
+        Tag.find_each do |tag_db|
```

Comment:
Tak, uzyj tej metody. Nie bedzie to jeszcze w 100% optymalne, ale juz bedzie lepiej. W kolejnym kroku, jak bedziesz miec wszystko poukladane juz jak trzeba, zamienimy to na wyciaganie z bazy od razu wszystkich tagow o `name` ktore zawiera sie w tablicy.

Jeszcze zeby doprecyzowac czemu `find_by` jest lepsze - `find_by` te petle, ktora wczesniej miales robi na poziomie bazy danych - sprtynie, optymalnie, uzywajac indexow (o ile masz jakies zdefiniowane). Generalnie po to jest baza, zeby byla zoptymalizowana pod szukanie, wiec jak chcesz cos wyszukac w bazie, to uzywasz interfejsu do szukania w bazie.